### DISCUSSION TITLE
Sign error 'path is too short'

### DISCUSSION THREAD

**Discussion by AKuzmichkin** [CoolBitX-Technology/coolwallet-signing-tools#71] [Q&A] [answered]
- Description
In script-tester, testing new script and get an error: returnCode: 6A0B, message: path is too short
script: 
03000202C707000000018DCAAC97000ADC07C0044E454152DAA0C7C00A0AD207CC05065052455353425554546F4E
argument: 0000000000000000000110000000616c65785f73636e2e746573746e6574009f6203c849d3b9bb51e46dba6f26452eae8b745efc613476ef59799a86e72ce3029e0b0e1e4d000011000000616c65785f73636e322e746573746e65742eb74a4a404dc36bcc88697f372434a9e1085707e6a334b063b450de3abcb9760100000003000000a1edccce1bc2d3000000000000

- Screenshots
![image](https://user-images.githubusercontent.com/101829495/158876693-318728a5-0b67-4d5f-8649-7230c08d5d59.png)

**Comment by hankliu418** [comment, marked as answer]
Would you please provide us this data in this command(80a2), thanks.

**Reply by AKuzmichkin** [reply]
ScriptData argAmount = sac.getArgument(10); // Amount
ScriptData argPayload = sac.getArgumentAll(); // Transaction payload

**Reply by AKuzmichkin** [reply]
And script:
        String script = new ScriptAssembler()
            .setCoinType(0x018d)
            .copyArgument(argPayload)
            .showMessage("NEAR")
            .showAmount(argAmount, 10)
            .showPressButton()
            .setHeader(HashType.SHA256, SignType.EDDSA)
            .getScript();

**Reply by hankliu418** [reply]
Do you add path before argument?
like.
`'15' + (await utils.getPath(COIN_TYPE, addressIndex)) + argument;`

**Reply by AKuzmichkin** [reply]
Nope. Thank you.
